source/renderer/modules/markdown-editor/plugins/render-citations.js

```diff
@@ -45,7 +46,7 @@
   // blocks, second alternative are the simple @ID-things, both recognised by
   // Pandoc citeproc.
   // citationRE is taken from the Citr library (the extraction regex)
```

Comment:
It might make sense to add this comment above the getCitationRE-func in the regexp-file, so that it's clear where it comes from (and, whenever I should change this RE, I know that I must update this as well)